browser/src/Services/QuickOpen.ts

```diff
@@ -55,46 +62,60 @@ export class QuickOpen {
                 const files = execSync(overriddenCommand.replace("${search}", ""), { cwd: process.cwd() })
                     .toString("utf8")
                     .split("\n")
-                this._showMenuFromFiles(files)
+                this._showMenuFromFiles(files, "file-text-o")
                 return
             } catch (e) {
                 Log.warn(`'${overriddenCommand}' returned an error: ${e.message}\nUsing default file list`)
             }
         }
 
         // Default strategy
-        //  If git repo, use git ls-files
+        //  If in exec directory, show bookmarks to change cwd to
+        //  Otherwise, If git repo, use git ls-files
         //  Otherwise, find all files recursively
         const openPromise = Git.isGitRepository()
             .then((isGit) => {
+                if (process.execPath === process.cwd()) {
```

Comment:
Also, we may want to move this check up, as it looks like it is only in the `Git` strategy. This could be problematic because, if the install directory is not controlled by git, we wouldn't hit this bookmark logic.